packages/python2.x86.vm/tools/chocolateyinstall.ps1

```diff
@@ -9,8 +9,8 @@ try {
     packageName    = ${Env:ChocolateyPackageName}
     fileType       = 'msi'
     silentArgs     = $installArgs
-    url            = 'https://www.python.org/ftp/python/2.7.15/python-2.7.15.msi'
-    checksum       = '1AFA1B10CF491C788BAA340066A813D5EC6232561472CFC3AF1664DBC6F29F77'
+    url            = 'https://www.python.org/ftp/python/2.7.18/python-2.7.18.msi'
```

Comment:
a change here also requires an update to the respective `.nuspec` file, i.e., the version string